packages/core/src/synchronizer.ts

```diff
@@ -138,11 +138,11 @@ export class UnirepSocialSynchronizer extends Synchronizer {
             'CommentSubmitted',
             event.data
         )
-        const _transactionHash = event.transactionHash
+        const transactionHash = event.transactionHash
         const commentId = event.transactionHash
         const postId = event.topics[2]
-        const _epoch = Number(event.topics[1])
-        const _epochKey = BigInt(event.topics[3]).toString(16)
+        const epoch = Number(event.topics[1])
+        const epochKey = BigInt(event.topics[3]).toString(16)
```

Comment:
I think we should store these as base 10, or base 16 with a `0x` prefix. If we do it like this we get a string that is base 16 with no `0x` prefix. We have no way to decode these types of strings properly.

We've been moving toward storing as base10 in other pr's.